examples/lang/tar.mcl

```diff
@@ -0,0 +1,20 @@
+$filepath = "/tmp/mgmt/mytar.tar"
+
+file "/tmp/mgmt/file1.txt" {
+	content => "julius caesar",
+	state => $const.res.file.state.exists,
+	Before => Tar["${filepath}"],
+}
+
+file "/tmp/mgmt/file2.txt" {
+	content => "hello world",
+	state => $const.res.file.state.exists,
+	Before => Tar["${filepath}"],
```

Comment:
Implementing AutoEdges for the `tar` resource would make this not necessary

Reply:
Will do.